src/pykeen/triples/generation.py

```diff
@@ -43,10 +45,13 @@ def generate_triples(
             relation: i
             for i, relation in enumerate(sorted(get_relations(rv)))
         }
-        rv = np.asarray([
-            [new_entity_id[h], new_relation_id[r], new_entity_id[t]]
-            for h, r, t in rv
-        ], dtype=int)
+        rv = torch.as_tensor(
```

Comment:
@mberr just curious - what's the difference between creating the tensor directly and using as_tensor?

Reply:
According to the doc of [torch.tensor](https://pytorch.org/docs/stable/generated/torch.tensor.html) and [torch.as_tensor](https://pytorch.org/docs/stable/generated/torch.as_tensor.html), `torch.as_tensor` is a more clever version which avoids creating copies when possible.

Also, at least in 1.6 `torch.tensor` did not have a proper type annotation which resulted in pylint warnings of `torch.tensor` not being callable, whereas `torch.as_tensor` didn't have this. This issue may have been fixed until then though.